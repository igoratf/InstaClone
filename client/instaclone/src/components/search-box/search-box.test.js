import React from 'react';
import { shallow } from 'enzyme';
import SearchBox from './search-box.component';

describe('Search Box component', () => {
  it('should render Search Box', () => {
    expect(shallow(<SearchBox />)).toMatchSnapshot();
  })
})