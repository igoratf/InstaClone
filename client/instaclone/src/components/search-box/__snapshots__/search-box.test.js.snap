// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Box component should render Search Box 1`] = `ShallowWrapper {}`;
